src/adapter/src/coord/sequencer/inner/peek.rs

```diff
@@ -531,96 +525,104 @@ impl Coordinator {
             determination,
             mut optimizer,
             explain_ctx,
-        }: PeekStageOptimizeMir,
+        }: PeekStageOptimize,
     ) {
```

Comment:
Can we rename this method to `peek_stage_optimize`?

Reply:
Done.